projects/framework-common-extensions/dialogs/standard_publisher_dialog.py

```diff
@@ -69,14 +72,21 @@ def pre_build_ui(self):
             QtCore.Qt.ScrollBarAlwaysOff
         )
 
-        # Create a main widget for the scroll area
-        self._scroll_area_widget = QtWidgets.QWidget()
-        scroll_area_widget_layout = QtWidgets.QVBoxLayout()
-        scroll_area_widget_layout.setContentsMargins(0, 0, 0, 0)
-        self._scroll_area_widget.setLayout(scroll_area_widget_layout)
+        index = self.layout().indexOf(self.tool_widget)
+        run_index = self.layout().indexOf(self.run_button)
+        if index != -1:  # Ensure the widget is found in the layout
+            # Remove the old widget from layout
+            self.layout().takeAt(index)
+            # Insert the new widget at the same position
+            self.layout().insertWidget(index, self._scroll_area)
```

Comment:
Might cause trouble with PySide6

Reply:
We will take a look when in PySide6 in case